libr/core/disasm.c

```diff
@@ -5483,11 +5488,12 @@ R_API int r_core_print_disasm(RPrint *p, RCore *core, ut64 addr, ut8 *buf, int l
 		////
 		int mi_type;
 		bool mi_found = ds_print_meta_infos (ds, buf, len, idx, &mi_type);
+		bool hasHint = false;
```

Comment:
not a blocker, but `s/hasHint/has_hint`.

Reply:
fixed